Geometry_Engine/Modify/RoundCoordinates.cs

```diff
@@ -0,0 +1,215 @@
+        /***************************************************/
+
+        [Description("Modifies a BHoM IGeometry's Points' coordinates to be rounded to the number of provided decimal places.")]
+        [Input("geometry", "The BHoM IGeometry to modify.")]
+        [Input("decimalPlaces", "The number of decimal places to round to, default 6.")]
+        [Output("geometry", "The modified BHoM IGeometry.")]
+        public static IGeometry IRoundCoordinates(this IGeometry geometry, int decimalPlaces = 6)
+        {
+            return RoundCoordinates(geometry as dynamic, decimalPlaces);
+        }
+
+        /***************************************************/
+
+        [Description("Modifies a BHoM ICurve's Points coordinates to be rounded to the number of provided decimal places.")]
+        [Input("curve", "The BHoM ICurve to modify.")]
+        [Input("decimalPlaces", "The number of decimal places to round to, default 6.")]
+        [Output("curve", "The modified BHoM ICurve.")]
+        public static ICurve IRoundCoordinates(this ICurve curve, int decimalPlaces = 6)
+        {
+            return RoundCoordinates(curve as dynamic, decimalPlaces);
+        }
+
+
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Modifies a BHoM Geometry Point to be rounded to the number of provided decimal places.")]
+        [Input("point", "The BHoM Geometry Point to modify.")]
+        [Input("decimalPlaces", "The number of decimal places to round to, default 6.")]
+        [Output("point", "The modified BHoM Geometry Point.")]
+        public static Point RoundCoordinates(this Point point, int decimalPlaces = 6)
+        {
+            return new Point
+            {
+                X = Math.Round(point.X, decimalPlaces),
+                Y = Math.Round(point.Y, decimalPlaces),
+                Z = Math.Round(point.Z, decimalPlaces),
+            };
+        }
+
+        /***************************************************/
+
+        [Description("Modifies a BHoM Geometry Line's end points to be rounded to the number of provided decimal places.")]
+        [Input("line", "The BHoM Geometry Line to modify.")]
+        [Input("decimalPlaces", "The number of decimal places to round to, default 6.")]
+        [Output("line", "The modified BHoM Geometry Line.")]
+        public static Line RoundCoordinates(this Line line, int decimalPlaces = 6)
+        {
+            return new Line()
+            {
+                Start = line.Start.RoundCoordinates(decimalPlaces),
+                End = line.End.RoundCoordinates(decimalPlaces)
+            };
+        }
+
+        /***************************************************/
+
+        [Description("Modifies a BHoM Geometry Arc's end points to be rounded to the number of provided decimal places.")]
```

Comment:
See above

Reply:
Changed to `start and end coordinates` to clarify which parameters of the `Arc` was modified